crates/constants/Cargo.toml

```diff
@@ -6,3 +6,6 @@ license = "GPL-3.0-only"
 homepage = "https://github.com/hulks/hulk"
 
 [dependencies]
+lazy_static = {workspace = true}
+serde = {workspace = true}
+serde_json = {workspace = true}
```

Comment:
This file is missing a terminating newline.

Reply:
done